homeassistant/components/hassio/config_flow.py

```diff
@@ -0,0 +1,26 @@
+"""Config flow for Home Assistant Supervisor integration."""
+import logging
+
+from homeassistant import config_entries
+
+from . import DOMAIN  # pylint:disable=unused-import
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for Home Assistant Supervisor."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_LOCAL_PUSH
+
+    async def async_step_user(self, user_input=None):
```

Comment:
If we don't activate the config flow in the manifest.json I don't think we need to define this step.